client/landing/gutenboarding/components/header/style.scss

```diff
@@ -25,19 +26,23 @@
 	}
 }
 
-.gutenboarding__header-wp-icon {
+.gutenboarding__header-wp-logo {
 	color: var( --studio-blue-90 );
+	display: flex;
 }
 
 .gutenboarding__header-section {
 	display: flex;
 	align-items: center;
 	font-size: 14px;
-	padding: 10px;
 }
 
 .gutenboarding__header-section-item {
 	margin-left: 10px;
+
+	&.gutenboarding__header-wp-logo {
+		margin-left: 24px - $grid-size; // ( 24 - header padding )
```

Comment:
Took me a minute to track this down, G2 is removing this variable:

https://github.com/WordPress/gutenberg/blob/c6df2f4207959a517884cabce26134e78f8bec6c/packages/base-styles/_variables.scss#L20-L33

Reply:
I guess sooner or later we are bound to come across these things since we copied the header styles over. 🤷‍♂